Dockerfile

```
@@ -141,16 +125,22 @@
 # Copy static assets from builder (if needed)
 COPY --from=builder /app/priv/static /app/priv/static
 
-# Copy chart-service from node builder
-COPY --from=node_builder /chart-service /app/chart-service
-
-# Copy runtime scripts and set executable permissions
-COPY scripts/start_with_db.sh scripts/db_operations.sh /app/bin/
+# Create environment variable debug script that will run before start.sh
+RUN mkdir -p /app/bin && echo '#!/bin/bash \n\
```

Hadolint (2.12.0)

[info] echo may not expand escape sequences. Use printf.

(SC2028)